### PR TITLE
Update BMLor403.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Updated textparts in ms_i2 according to https://github.com/BetaMasaheft/Works/pull/1282.

### CODE REVIEW

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 1]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                                 <locus from="49r" to="78v"/>
                                 <title ref="LIT4310Prayer"/>
-                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ። ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ስብሐት፡ ወጸሎተ፡ ወወድሶ፡ ወመለኮት፡ ወተማኅፀኖ፡ በአስማተ፡
-                                    ለስርየተ፡ ኃጢአት። ኦእግዚኦ፡ እግዚአ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ በአስማት። አሌፍ። ካፍ። ከዋፍ። ኤፍ።</incipit>
-                                <explicit xml:lang="gez">አንተ፡ ውእቱ፡ ክረምት፡ ወሐጋየ፡ ለከ፡ ስብሐት፡ ወአኰቴት፡ በምድር፡ ወበሰማይ፡ ለእግዚእየ፡ ወአምላኪየ፡ ኢየሱስ፡ ክርስቶስ።
-                                    ኵሎ፡ ጊዜ፡ ይእዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን። ወአሜን። ለይኩን፡ ለይኩን።</explicit>
+                                <msItem xml:id="ms_i2.1">
+                                    <locus target="#49r"/>
+                                    <title ref="LIT4310Prayer#supplication"/>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ስብሓት፡ ወጸሎ<sic resp="CH">ተ</sic>፡ ወወድሶ፡ መለኮት፨
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:58]: ወመለኮት፨ ?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 2]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                                 <title ref="LIT4310Prayer"/>
-                                <incipit xml:lang="gez">በስመ፡ አብ፡ ወወልድ። ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ። ስብሐት፡ ወጸሎተ፡ ወወድሶ፡ ወመለኮት፡ ወተማኅፀኖ፡ በአስማተ፡
-                                    ለስርየተ፡ ኃጢአት። ኦእግዚኦ፡ እግዚአ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ በአስማት። አሌፍ። ካፍ። ከዋፍ። ኤፍ።</incipit>
-                                <explicit xml:lang="gez">አንተ፡ ውእቱ፡ ክረምት፡ ወሐጋየ፡ ለከ፡ ስብሐት፡ ወአኰቴት፡ በምድር፡ ወበሰማይ፡ ለእግዚእየ፡ ወአምላኪየ፡ ኢየሱስ፡ ክርስቶስ።
-                                    ኵሎ፡ ጊዜ፡ ይእዜኒ፡ ወዘልፈኒ፡ ወለዓለመ፡ ዓለም፡ አሜን። ወአሜን። ለይኩን፡ ለይኩን።</explicit>
+                                <msItem xml:id="ms_i2.1">
+                                    <locus target="#49r"/>
+                                    <title ref="LIT4310Prayer#supplication"/>
+                                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ስብሓት፡ ወጸሎ<sic resp="CH">ተ</sic>፡ ወወድሶ፡ መለኮት፨ 
+                                        ወተማኅፅኖ፡ በአስማ<sic resp="CH">ተ</sic>፡ ለስርየተ፡ ኀጢአት።</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:59]: ኃጢአት፨ ?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 3]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.2">
+                                    <locus from="49r" to="49v"/>
+                                    <title ref="LIT4310Prayer#first"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡ አሌፍ። ካፍ።</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:64]: አስማት፨ አአሌፍ፨ ?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 4]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.3">
+                                    <locus from="49v" to="50v"/>
+                                    <title ref="LIT4310Prayer#second"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ ዕቀበኒ፡ እምሥገርቱ፡</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:69]: እመሥገርቱ

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 5]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.9">
+                                    <locus from="56v" to="57r"/>
+                                    <title ref="LIT4310Prayer#eighth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ ዘዚአከ፡ ሣህል፡ እስመ፡ በኀቤከ፡ ኵሎ፡ ይትከሀልከ፡</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.10">
+                                    <locus from="57r" to="59r"/>
+                                    <title ref="LIT4310Prayer#ninth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አድኅነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት። ዳሌጥ፡ ወሄት። አድኅነኒ፡ ዘአድኅንኮ፡</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:104]: ዘአድኀንኮ

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 6]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.11">
+                                    <locus from="59r" to="59v"/>
+                                    <title ref="LIT4310Prayer#tenth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ ግብርከ፡ ወምግባርከ፡</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.12">
+                                    <locus from="59v" to="60r"/>
+                                    <title ref="LIT4310Prayer#eleventh"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡ ሄ፡ ለሄ። ወለለሄ። እንዘ፡ ሥልጣንሰ፡ ምሉእ፡</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:114]: I read ወለላሄ

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 7]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.15">
+                                    <locus from="62v" to="64r"/>
+                                    <title ref="LIT4310Prayer#fourteenth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡ ዛይ፡ አዶናይ፡ ፈጣሪሁ፡ ለሰማይ።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.16">
+                                    <locus from="64v" to="65r"/>
+                                    <title ref="LIT4310Prayer#fifteenth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡ ጤት፡ ወሔት፡ ወሳምኬት፡ ለምንተ፡ ገዳፍከኒ፡ አድኅነኒ፡
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:135]: ገደፍከኒ, no?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 8]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                    <locus from="68r" to="68v"/>
+                                    <title ref="LIT4310Prayer#eighteenth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት። ኖን። አፍርዮን። ምክራድዮን። እምነቅዐ፡ ሕይወት፡ 
+                                        ከመ፡ ንርወዮን።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.20">
+                                    <locus from="68v" to="70r"/>
+                                    <title ref="LIT4310Prayer#nineteenth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ዘዚአከ፡ አስማት፡ ሜ፡ ተኖሜ፡ ዘከለልኮ፡ ለታቦር፡ በጊሜ፡ ተማኅፀንኩ፡ በደብረ፡ 
+                                        ቍስቋም፡ በዘ፡ ነበርከ፡ ምስለ፡ እምከ፡ ቀዳሚ፡ በአሣእኒከ፡ ዘጸረት፡ ሰሎሜ።</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:159]: ዘጾረት፡ ?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 9]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </msItem>
+                                <msItem xml:id="ms_i2.21">
+                                    <locus from="70r" to="73r"/>
+                                    <title ref="LIT4310Prayer#twentieth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ ለከ፡ ኦዋሕድ፡ ዘቆመ፡ ቅድመ፡ ጲላጦስ፡</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.22">
+                                    <locus from="73r" to="73v"/>
+                                    <title ref="LIT4310Prayer#twentyfirst"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በስምከ፡ ዔ፡ ጥናዔ፡ ጰናዔ፡ ዘሰአልነ፡ ኀቤከ፡ ኩን፡ ሰማዔ።</incipit>
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:169]: ጥናዔ፨ (or ጥናዔ።)

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 10]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <msItem xml:id="ms_i2.24">
+                                    <locus from="74v" to="75v"/>
+                                    <title ref="LIT4310Prayer#twentythird"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አነኒ፡ ተማኅፀንኩ፡ በዝንቱ፡ ስምከ፡ ሬሲ፡ ትስ<pb n="75r"/>እለከ፡ ዘዚኣየ፡ ነፍስ፡ አበሳየ፡ ደምስስ፡ 
+                                        እንተ፡ ጽንዕት፡ ነፍሰ፡ በጠለ፡ ምሕረትከ፡ አርሕስ።</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.25">
+                                    <locus from="75v" to="77r"/>
+                                    <title ref="LIT4310Prayer#twentyfourth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አድኅ<sic>ኒ</sic>ኒ፡ ተማኅፀንኩ፡ ዘዚአከ፡ አስማት፡ ሳን። ኦን፡ ፍኑን። ዝንቱ፡ ዘተብህለ፡ እብን፡ ምኑን፡
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:185]: ኑን rather than ኦን ?

File: FlorenceBML/BMLor403/BMLor403.xml [hunk 11]
```diff
@@ -52,10 +52,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <msItem xml:id="ms_i2.25">
+                                    <locus from="75v" to="77r"/>
+                                    <title ref="LIT4310Prayer#twentyfourth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ እግዚኦ፡ አድኅ<sic>ኒ</sic>ኒ፡ ተማኅፀንኩ፡ ዘዚአከ፡ አስማት፡ ሳን። ኦን፡ ፍኑን። ዝንቱ፡ ዘተብህለ፡ እብን፡ ምኑን፡ 
+                                        ውስተ፡ ርእሰ፡ ማእዝንት፡</incipit>
+                                </msItem>
+                                <msItem xml:id="ms_i2.26">
+                                    <locus from="77r" to="78v"/>
+                                    <title ref="LIT4310Prayer#twentyfifth"/>
+                                    <incipit xml:lang="gez">ኦእግዚኦ፡ ነጽር፡ ውስተ፡ ረዲኦትየ፡ እግዚኦ፡ አፍጥን፡ ረዲአትየ፡ ይትኃፈሩ፡ ወይኅሠሩ፡ ኵሎሙ፡ ጸላእትየ፡ ለይግብኡ፡
```
Comment (karljonaskarlsson) [inline FlorenceBML/BMLor403/BMLor403.xml:191]: Maybe ረዲኦትየ also the second time? Alternatively, I would put a <sic>

**Review by karljonaskarlsson — APPROVED** [review]
It's very interesting, this text! Thanks for adding all the incipits :)

**Review by eu-genia — APPROVED** [review]
thank you!

**Pull request merged**